python/smt_switch_imp.pxd

```diff
@@ -0,0 +1,125 @@
+from libc.stdint cimport int32_t, int64_t, uint32_t, uint64_t
+from libcpp.memory cimport shared_ptr
+from libcpp.string cimport string
+from libcpp.unordered_map cimport unordered_map
+from libcpp.vector cimport vector
+
+from enums cimport SortKind
+from enums cimport PrimOp
+
+ctypedef shared_ptr[AbsSort] Sort
+ctypedef shared_ptr[AbsTerm] Term
+ctypedef shared_ptr[AbsSmtSolver] SmtSolver
+ctypedef vector[Sort] SortVec
+ctypedef vector[Term] TermVec
+# ctypedef unordered_map[Term, Term] UnorderedTermMap
```

Comment:
Not sure what this is/was for, but remove this comment if it isn't necessary.

Reply:
Ope, good catch! Removed with c534d7afab3064227bc999ae41bd44e9cbf5370b. If you're curious, I was just trying to declare it like I did for all the others, but it didn't work because it requires a third template argument for the hash function. Instead, I just declared the C++ UnorderedTermMap class and it's emplace method (this is used for `substitute`)